src/core/PWSfileV3.cpp

```diff
@@ -1001,6 +1001,8 @@ int PWSfileV3::ReadHeader()
     FILE *fd = pws_os::FOpen(filename.c_str(), _T("rb"));
 
     ASSERT(fd != nullptr);
+    if (fd == nullptr)
+      return false;
```

Comment:
Is a similar change also useful for v4 and v2?

Reply:
They both call CheckPasskey which already contains this check (for both versions).